src/Components/common/BigTile/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  Container,
+  TileImage,
```

Comment:
Uważam że elementy zawarte w komponencie nie muszą być nazywane od nazw komponentu tylko porostu czym są w całej strukturze, czyli jak masz to powinno wystarczyć samo  itd.

Reply:
Ok, zmienie to w takim razie, chcialem zeby to bylo bardziej przejrzyste, ale moze wystarczy, ze bedzie odpowiednia nazwa komponentu :P

Reply:
Tak :)

Reply:
Słuszna uwaga :) też jestem za tym, żeby jak najprościej nazywać elementy.